docs/Concepts/Privacy/PrivacyEnhancements.md

```diff
@@ -5,101 +5,170 @@ description: Privacy Enhancements feature prevents nodes from modifying contract
 
 # Privacy Enhancements
 
-## Overview
+From GoQuorum v20.10.0, two privacy enhancements are included to prevent state divergence:
 
-Before GoQuorum v20.10.0 private contract implementations had to include access controls to protect against other network participants from updating the state. The privacy enhancements feature has been developed to prevent such "non-party" interaction without additional access controls. This is achieved with the introduction of the `ACOTH` which is short for "Affected Contract's Original Transaction's encrypted payload Hash".
-In order to transact with a contract a node must prove it has access (has the encrypted payload and the ability to decrypt it) to the contract's originating transaction.
-Two flavours of privacy enhanced contracts have been introduced: a) Counter party protection and b) Private state validation.
+* Counter party protection
+* Private state validation
 
-## Counter Party Protection (PP)
+## Counter party protection
 
-This flavor prevents non-party interaction on a private contract but allows state divergence. That is, it will allow nodes to maintain different state through private transaction to "self" or "subset of nodes".
+Counter party protection prevents non-participants sending a transaction to a
+private contract. For example, a private contract is deployed between node 1 and 2. Without counter
+party protection, if node 3 discovers the private contract address, node 3 can send a transaction with
+`privateFor` set to node 2. The transaction changes the private state of node 1 but not node 2 and
```

Comment:
Should this be "The transaction changes the private state of node 2 but not node 1” as the privateFor is set to 2?

Reply:
Hi @MadelineMurray, may I suggest the following change: ```Counter party protection prevents non-participants sending a transaction to a
private contract. For example, a private contract is deployed between node 1 and 2. Without counter
party protection, if node 3 discovers the private contract address, node 3 can send a transaction with
`privateFor` set to node 2. This transaction will not be applied on node 3 as it does not party to the private contract. However, the state change will happen in node 2```

Reply:
We want to stress on a nonparticipant sending a transaction to participants. Even with party protection the state divergence can happen across participants. For example contract deployed between 1,2 and 3 and then private for between say 1 & 2

Reply:
Updated